### PR TITLE
Issue #1007: Run postfix notification scripts with dedicated user openmediavault-notify

### PR DESCRIPTION
**Pull request by votdev**
Fixes: https://github.com/openmediavault/openmediavault/issues/1007

Signed-off-by: Volker Theile <votdev@gmx.de>

- [x] References issue
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

File: deb/openmediavault/usr/share/openmediavault/notification/sink.d/README [hunk 1]
```diff
@@ -13,9 +13,12 @@ OMV_NOTIFICATION_SUBJECT
 OMV_NOTIFICATION_DATE
 OMV_NOTIFICATION_MESSAGE_FILE
 
-Additionaly the scripts/executables are executed with the following command
+Additional the scripts/executables are executed with the following command
```
Comment (djmattyg007) [inline deb/openmediavault/usr/share/openmediavault/notification/sink.d/README:16]: I think should be changed to `Additionally`

File: deb/openmediavault/debian/openmediavault.postinst [hunk 2]
```diff
@@ -289,6 +298,9 @@ case "$1" in
 			LANG=C.UTF-8 omv-salt deploy run --no-color --quiet apt cronapt || :
 		fi
 		if dpkg --compare-versions "$2" lt-nl "5.6.4"; then
+		  useradd --system --user-group --no-create-home \
+		    --shell '/usr/sbin/nologin' openmediavault-notify || :
+			omv_module_set_dirty postfix
```
Comment (djmattyg007) [inline deb/openmediavault/debian/openmediavault.postinst:303]: The indentation is inconsistent here.

File: deb/openmediavault/usr/share/openmediavault/notification/sink.d/README (hunk 1 above)
Comment (votdev) [reply deb/openmediavault/usr/share/openmediavault/notification/sink.d/README]: Thx, didn't check the result after the IDE has changed that typo automatically. Now changed to `Additionally` manually.

File: deb/openmediavault/debian/openmediavault.postinst (hunk 2 above)
Comment (votdev) [reply deb/openmediavault/debian/openmediavault.postinst]: Another strange IDE behavior. The IDE is mixes tabs and blanks wildly.

**Pull request merged**